editor/filesystem_dock.cpp

```diff
@@ -1338,10 +1335,15 @@ void FileSystemDock::_search_changed(const String &p_text) {
 
 void FileSystemDock::_rescan() {
 
+	print_line("call rescan!");
 	_set_scanning_mode();
 	EditorFileSystem::get_singleton()->scan();
 }
 
+void FileSystemDock::rescan() {
```

Comment:
Why not simply rename `_rescan` to `rescan`?

Though I'm sure if rescanning everything just to get files visually sorted in the dock when saving is the right approach, sounds overkill.

Reply:
I didn't rename the function because of this:
https://github.com/godotengine/godot/blob/3ad9e4740c35ec20fdd6942f13047ebd53ea203e/editor/filesystem_dock.cpp#L1738
I wasn't sure if I should rename both there too, I believe that might break compatibility a bit, which is why I decided to leave the pull request as is.

Changing this to only sort the files when a new file has been saved in the visible directory is a better approach, but I haven't looked into it yet. I figured I wanted to fix that function name properly first.